packages/golden_toolkit/README.md

```diff
@@ -69,9 +69,9 @@ The output of this test will be this golden file:
 In order to test with different font sizes, do:
 
 ```dart
-        ..addTest('Regular font size', widget)
-        ..addTestWithLargeText('Large font size', widget, maxTextSize: 2.0)
-        ..addTestWithLargeText('Largest font', widget, maxTextSize: 3.2);
+        ..addScenario('Regular font size', widget)
+        ..addTextScaleScenario('Large font size', widget, textScaleFactor: 2.0)
```

Comment:
I was trying to eliminate the "test" from the naming. as I looked at it, I realized there may also be cases where someone may want to shrink the font below 1.0 in which case "Large" is misleading. textScaleFactor felt most appropriate